bestiary/core/schema.py

```diff
@@ -246,9 +273,91 @@ class Meta:
     )
 
 
+class EcosystemPaginatedType(AbstractPaginatedType):
+    """Type returning paginated results of EcosystemType objects"""
+
+    entities = graphene.List(EcosystemType)
+    page_info = graphene.Field(PaginationType)
+
+
+class AddEcosystem(graphene.Mutation):
+    """Mutation which adds a new Ecosystem into the database"""
```

Comment:
Maybe we shouldn't talk about `database` and say `on the registry`, `on the system`,`on the service`, and so on.